### PR TITLE
Added more 8ball commands.

### PR DESCRIPTION
**Pull request by cgunston**
For https://github.com/RPCS3/discord-bot/issues/117

### CODE REVIEW

File: CompatBot/Commands/Misc.cs [hunk 1]
```diff
@@ -25,6 +25,8 @@ internal sealed class Misc: BaseCommandModuleCustom
             "Don't be an idiot. YES.", "Mhm!", "Many Yes", "Yiss", "Sir, yes, Sir!", 
             "Yah!", "Ja", "Umu!", "Make it so", "Sure thing",
             "Certainly", "Of course", "Definitely", "Indeed",
+            "Much yes", "Make it so", "Consider it done", "Totally", "You bet",
```
Comment (13xforever) [inline CompatBot/Commands/Misc.cs]:
```suggestion
            "Much yes", "Consider it done", "Totally", "You bet",
```
plz no duplicates

File: CompatBot/Commands/Misc.cs [hunk 2]
```diff
@@ -25,6 +25,8 @@ internal sealed class Misc: BaseCommandModuleCustom
...
+            "Sure", "Yup", "Yep", "Okie dokie", "Certainly", "Yarp",
```
Comment (13xforever) [inline CompatBot/Commands/Misc.cs]:
```suggestion
            "Yup", "Yep", "Certainly", "Yarp",
```
"okie dokie" is not really a positive answer, but an acknowledgement on the bot's part, so it doesn't really fit
also "yup" is already present, but in a slightly different form

**Comment by cgunston** [comment]
Hi 13xforever

Thanks for the comments, I'm still just getting started at the moment.

I've resolved my commits based on your comments. Does that seem ok now?

Thanks!

**Comment by 13xforever** [comment]
@cgunston can you squash everything to one commit? I'd prefer this to be an atomic change, and I don't want to use autosquash on github as it will change the author

**Comment by cgunston** [comment]
@13xforever Apologies what would be the best way to achieve that?

**Comment by 13xforever** [comment]
if you don't know then it's ok, I'll just merge as is

**Comment by cgunston** [comment]
@13xforever Thank you! 👍

**Pull request merged**